integration-tests/appsec/endpoints-collection.spec.js

```diff
@@ -73,16 +52,57 @@ describe('Endpoints collection', () => {
       { method: 'HEAD', path: '/api/sub/deep' },
       { method: 'POST', path: '/api/sub/deep/:id' },
 
-      // Wildcard routes
-      { method: 'GET', path: '/wildcard/*' },
-      { method: 'HEAD', path: '/wildcard/*' },
-      { method: 'GET', path: '*' },
-      { method: 'HEAD', path: '*' },
-
       { method: 'GET', path: '/later' },
       { method: 'HEAD', path: '/later' },
     ]
 
+    if (framework === 'fastify') {
+      expectedEndpoints.push(
+        { method: 'OPTIONS', path: '/users/:id?' },
+
+        // Route with regex - not supported in express5
+        { method: 'DELETE', path: '/regex/:hour(^\\d{2})h:minute(^\\d{2})m' },
+
+        { method: 'OPTIONS', path: '/users/:id?' }, // Added with addHttpMethod
+        { method: 'MKCOL', path: '/example/near/:lat-:lng/radius/:r' }, // Added with addHttpMethod
+
+        // All supported methods route
+        { method: 'GET', path: '/all-methods' },
+        { method: 'HEAD', path: '/all-methods' },
+        { method: 'TRACE', path: '/all-methods' },
+        { method: 'DELETE', path: '/all-methods' },
+        { method: 'OPTIONS', path: '/all-methods' },
+        { method: 'PATCH', path: '/all-methods' },
+        { method: 'PUT', path: '/all-methods' },
+        { method: 'POST', path: '/all-methods' },
+        { method: 'GET', path: '/wildcard/*' },
+        { method: 'HEAD', path: '/wildcard/*' },
+
+        // Wildcard routes
+        { method: 'GET', path: '*' },
+        { method: 'HEAD', path: '*' }
```

Comment:
restoring the previous order
```suggestion
        // Wildcard routes
        { method: 'GET', path: '/wildcard/*' },
        { method: 'HEAD', path: '/wildcard/*' },
        { method: 'GET', path: '*' },
        { method: 'HEAD', path: '*' }
```

Reply:
Merge conflict issue, fixed